lib/api.js

```diff
@@ -2,9 +2,15 @@
 
 const Core = require('./core');
 const Extensions = require('./extensions');
+const Apps = require('./apps');
+const Batch = require('./batch');
+const RBAC = require('./rbac');
```

Comment:
Our convention has been to avoid consecutive caps (e.g., `apiUrl`, not `apiURL`).  So this would be `Rbac` not `RBAC`.  `Rbac` looks funny, I know, but it follows the convention we're currently using.